packages/css/stories/Button.js

```diff
@@ -0,0 +1,19 @@
+import '../src/components/button.css';
+
+export const createButton = ({
+  mode = 'secondary',
+  size = 'medium',
+  disabled = false,
+  label,
+  onClick,
+}) => {
+  const btn = document.createElement('button');
+  btn.type = 'button';
+  btn.innerText = label;
+  btn.addEventListener('click', onClick);
+
+  btn.className = ['button-base', 'button-master', mode, size].join(' ');
```

Comment:
I think it would be easier for end users if they don't have to add both `button-base` and `button-master` classes, but just one class 

If they would always have to use both classes together then it should be just one CSS class name